types/storybook__addon-knobs/react.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@storybook/addon-knobs/react' {
```

Comment:
Remember to add these to "files" in `tsconfig.json`.

Reply:
a65fd408d25c20942f489d136079bef4ba879518